### DISCUSSION TITLE
GetUserInfo: http Post or Get?

### DISCUSSION THREAD

**Discussion by WZRPW** [slack-go/slack#1007] [Q&A] [answered]
The Slack-api Golang code uses the HTTP post for GetUserInfo:
https://github.com/slack-go/slack/blob/792f7ed7aeae758f493239a9c50bb5d18eefc46e/users.go#L239

But Slack API doc says it is HTTP Get:
https://api.slack.com/methods/users.info
I am confused. Which one is incorrect? Thanks,

**Comment by seratch** [comment, marked as answer]
Technically, both HTTP methods work! This is the same for other endpoints too. The preferred HTTP method in the document is a recommendation.
